env-files/torch/generic_torch.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
-
-# ENV VARIABLES:
+#ENV VARIABLES:
```

Comment:
minor detail: space after `#` and before `ENV VARIABLES:` makes the comment more readable